src/Http/WebUtilities/src/FileBufferingReadStream.cs

```diff
@@ -349,6 +361,39 @@ public override void Flush()
             throw new NotSupportedException();
         }
 
+        public override Task CopyToAsync(Stream destination, int bufferSize, CancellationToken cancellationToken)
```

Comment:
How sure are you that you fixed the CopyToAsync issue from #24032? The issue there was that the bufferSize parameter was being set to 1 by the non-virtual `CopyToAsync(Stream)` if the current stream length was 0 (nothing buffered yet).

Reply:
Oops